app/models/work.rb

```diff
@@ -0,0 +1,21 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+
+  validates :title, presence: true
+  validates :category, presence: true
+  validates :creator, presence: true
+  validates :description, presence: true
+
```

Comment:
This works, but I suggest you could make a model method called `upvote` which would take a user as an argument and create a vote for the current work.